nexus/db-queries/src/db/datastore/sled.rs

```diff
@@ -103,6 +137,22 @@ impl DataStore {
         Ok(all_sleds)
     }
 
+    #[cfg(test)]
```

Comment:
Sorry if this is a dumb question, but can we not use `LookupPath::new(...).sled_id(...)` for this?

Reply:
We absolutely can! I'd somehow forgotten about this. Will fix.